File: flower/src/main/java/com/kharitonov/flower/entities/BasketEntity.java
package com.kharitonov.flower.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "basket")
public class BasketEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    @OneToMany(cascade = CascadeType.ALL)
    @JoinColumn(name = "basket_id")
    private List<FlowerEntity> flowers = new ArrayList<>();
}

File: src/main/java/by/hghgrtut/shop/entities/BasketEntity.kt
package by.hghgrtut.shop.entities

import java.util.ArrayList
import javax.persistence.*

@Entity
@Table(name = "basket")
class BasketEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private val id: Long? = null,

    @OneToMany(cascade = CascadeType.ALL)
    @JoinColumn(name = "basket_id")
    private val items: List<ItemEntity> = ArrayList<ItemEntity>()
)